ldms/src/ldmsd/ldmsd_config.c

```diff
@@ -331,8 +385,23 @@ int ldmsd_load_plugin(char* cfg_name, char *plugin_name,
 			errno, plugin_name);
 		goto err;
 	}
+	sprintf(&log_name[strlen(log_name)], "%s.%s", api->name, cfgobj->name);
```

Comment:
I'd certainly hope all that fits in 1024 characters, but we should probably have bounds checking anyway.